test/language/statements/class/subclass/class-definition-null-proto-missing-return-override.js

```diff
@@ -38,4 +38,4 @@ class Foo extends null {
 
 var foo = new Foo();
```

Comment:
See https://github.com/tc39/ecma262/issues/543#issuecomment-210191577 for the discussion.